packages/core/tdoptions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "excludeNotExported": true,
+  "exclude": "lib, node_modules",
+  "experimentalDecorators": true,
+  "ignoreCompilerErrors": true,
+  "mode": "file",
+  "out": "./docs/core",
```

Comment:
why repeat the package name ? we're already in it ...
maybe docs/typedoc ? since we may have other docs